### DISCUSSION TITLE
Unable to Add Free Collaborators

### DISCUSSION THREAD

**Discussion by csc5k** [community/community#23317] [New to GitHub]
<p>I received an email about being to add unlimited free collaborators to private repos. I have a private repo within my organization but it’s only allowing me to have 1 collaborator. When I try to add another collaborator,  I get this error: You must purchase at least one more seat to add this user as a collaborator.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/csc5k">@csc5k</a>,</p>
<p>Unlimited private repo collaborators are now available for free for organizations subscribed to GitHub Free. However, for organizations subscribed to GitHub Team or GitHub Enterprise, all members and collaborators must still occupy a paid seat. If all of your org’s seats are currently filled, you’ll need to <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-your-github-subscription#adding-seats-to-your-organization" rel="nofollow noopener">increase the organization’s paid seat count</a> to resolve that error message.</p>

**Reply by bit-mlang** [reply]
What about if two different companys work on private repos. So there is Company A und Company B. Both pay for Team or Enterprise. Now Company B whats to work on a private repo of Company A. Is that possible without extra costs?

**Reply by MartinBarker** [reply]
So when you're in the 'free' plan you get unlimited repo collaborators, but once you pay for the Team or Enterprise plan, this feature goes away?

Why does the GitHub pricing page say `Everything included in Free, plus...` if this is a lie?
https://github.com/pricing
![image](https://github.com/user-attachments/assets/2d2b317f-7ddd-432e-839b-aa387208c2a7)

**Comment by vicaya** [comment]
Your pricing plan clearly says "Everything included in Free, plus.." for Team. You should at least add an `*` and footnote about the consequence of converting Free to Team...

**Reply by semiessessi** [reply]
I think this is a severe enough misdirection to constitute illegal sales activity.

I've just looked and the message is still the same. This should be corrected with urgency, even if the mistake is probably innocent and the resulting potential liabilities miniscule.

**Reply by lacymorrow** [reply]
This is shady, still the case

**Reply by MartinBarker** [reply]
Still the case

**Reply by mamoraw** [reply]
just experienced the same 😒

**Reply by AlexanderLLid** [reply]
Experienced the same? wtf is this?

**Reply by jonassiewertsen** [reply]
This is odd. I did just ran into this. 

I have a paid team plan and want to invite a customer of mine into the projects overview, so they can see what we are working on. 
Paying for a full seat, just so they can see whats going on? They are not coders and will probably login once a week. 

It's simply not what I expected.

I would expect that it is possible to invite members without paying extra, as outside collaborators to projects (and into repos).
This does make us as a team considering using the free version again or switching the Plattform.

**Reply by TohaRhymes** [reply]
Hi! Have you solved the problem somehow?

**Reply by jonassiewertsen** [reply]
Nope. Switched to the free version again ...
